WordPress/gutenberg
# [RNMobile][Embed block] Add the top 5 specific embed blocks to the Block inserter list

`gutenberg-mobile` PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/3995





## Description


Add the top 5 specific embed blocks to the Block inserter list ~~and use the colored version of the icons~~:
- YouTube
- Twitter
- WordPress
- ~~Instagram~~ (**NOTE: This block is not added yet, it will when we tackle [this issue](https://github.com/wordpress-mobile/gutenberg-mobile/issues/3951)**)
- Vimeo

**EDIT:** As mentioned in [this comment](https://github.com/WordPress/gutenberg/pull/34967#issuecomment-930381158), the colored versions of the icons won't be used.

## How has this been tested?



1. Open a post/page.
2. Tap on the ➕ button to open the block inserter list.
3. Scroll down the block list.
4. Observe that the previously mentioned providers have their own block in the list.

## Screenshots 

**iOS:**

Light mode|Dark mode
--|--
<img src=https://user-images.githubusercontent.com/14905380/135317629-ad76dbb8-2d4b-4cbf-aca7-6fbe7ae094a9.png>|<img src=https://user-images.githubusercontent.com/14905380/135317633-37ed8152-0ccc-4113-b91c-d02685f73ea3.png>

**Android:**

Light mode|Dark mode
--|--
<img src=https://user-images.githubusercontent.com/14905380/135317915-b1018b1a-7332-4ff8-af62-56081644fd80.jpg>|<img src=https://user-images.githubusercontent.com/14905380/135317921-9b1b0300-d126-4fb4-9ce3-af47e9522f7f.jpg>

## Types of changes




New feature

## Checklist:
